### PR TITLE
Make loader pip installable

### PR DESCRIPTION
**Pull request by jessebrennan**
The loader is now pip installable! give it a try here https://pypi.org/project/cgp-dss-data-loader/

I will do a real release once this PR is approved. Also we need to discuss the release process. I'm assuming that's a tagged merge commit to master from develop? Need to talk with @mikebaumann.

Also currently it is pushed to my PyPI account, but that can change if need be.gg

┆Issue is synchronized with this [JIRA Task](https://ucsc-cgl.atlassian.net/browse/LOAD-29)
┆Project Name: CGP-Data Loader
┆Issue Number: LOAD-29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,65 +2,60 @@
...
-   `git clone https://github.com/DataBiosphere/cgp-dss-data-loader.git`
+1. Run:
 
-2. Go to the root directory of the cloned project:
-   
-   `cd cgp-dss-data-loader`
+   `pip3 install cgp-dss-data-loader`
 
-3. Run (ideally in a new [virtual environment](https://docs.python.org/3/tutorial/venv.html)):
+## setup for development
```
Comment (natanlao) [inline README.md]: Is there a reason that all of this is lowercase now?

**Review by natanlao — APPROVED** [review]
Looks good to me!

File: README.md (hunk 1 above)
Comment (jessebrennan) [reply README.md]: whoops!

**Comment by natanlao** [comment]
Take a look at the entire document - case is flipped for a good section of it

**Pull request merged**